src/blocks/FilterBlock/FilterBlock.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {useMemo, useState} from 'react';
+
+import {block} from '../../utils';
+import {FilterBlockProps as FilterBlockParams} from '../../models';
+import {Row, Col} from '../../grid';
+import {BlockHeader, AnimateBlock} from '../../components';
+import ButtonTabs, {ButtonTabsItemProps} from '../../components/ButtonTabs/ButtonTabs';
+
+import FilterBlockContext, {FilterData} from './FilterBlockContext';
+
+import './FilterBlock.scss';
+
+export interface FilterableProps extends Omit<FilterBlockParams, 'child'> {
+    children?: React.ReactNode;
+}
+
+const b = block('filter-block');
+const DEFAULT_ALL_TAG_TITLE = 'All';
+
+const FilterBlock: React.FC<FilterableProps> = ({
+    title,
+    description,
+    filterTags,
+    tagSize,
+    allTag,
+    animated,
+    children,
+}) => {
+    const tabButtons = useMemo(() => {
+        const allButton: ButtonTabsItemProps | undefined = allTag
+            ? {id: '', title: typeof allTag === 'boolean' ? DEFAULT_ALL_TAG_TITLE : allTag}
```

Comment:
Why `id` is empty string? Don't understand what happens if `allTag` is true? Should you create a styoy when allTag is true or Default?

Reply:
The actual API of the ButtonTabs component requires that id will be specified.
In case of the `true` value of `allTag` the component adds the All tag with the default label.
Sure, will add a story.

Reply:
Maybe then we have to set id as not necessary prop in  ButtonTabs?

Reply:
I'm not sure that it is good idea to make it unnecessary, but I believe that making it nullabe will do.

Reply:
I see that now `id` can be null, but here still an empty string